src/navigators/RootNavigator.tsx

```diff
@@ -90,6 +93,15 @@ export const RootNavigator = () => {
     const initializeApp = async (
       postInitializeCallback?: () => void,
     ): Promise<void> => {
+      const deviceCompromised = isDeviceJailbroken();
+      setIsJailbroken(!deviceCompromised);
```

Comment:
is this intended to set it like `!deviceCompromised` instead of `deviceCompromised` ?

on line 159 it's also using `!isJailbroken` instead of `isJailbroken`

if we remove the `!` on both places I believe it'd look a bit more intuitive. Wdyt?